src/client/App.scss

```diff
@@ -124,3 +124,16 @@ footer {
 .was-validated .unchecked .invalid-feedback {
   display: block;
 }
+
+// For help text on pages and components
+.help-text {
+  font-size: 20px;
```

Comment:
In other places we use rem for sizing. I think this would be 1.25rem?

Reply:
In her request, Tina specified 20px for the font size.